src/nrniv/nvector_nrnparallel_ld.cpp

```diff
@@ -15,19 +15,20 @@
  * of the NVECTOR package.
  * -----------------------------------------------------------------
  */
+
+#define SUNTRUE  true
+#define SUNFALSE false
 #include <../../nrnconf.h>
 #include <hocassrt.h>
 
 #include <stdio.h>
 #include <stdlib.h>
 
-#include <nrnmpiuse.h>
-#include <nrnmpidec.h>
-extern int nrnmpi_numprocs;
-
 #include "nvector_nrnparallel_ld.h"
-#include "sundialsmath.h"
-#include "sundialstypes.h"
+#define MPI_Comm int  // or perhaps get rid of them all
```

Comment:
nitpick - just curious about this

Reply:
I am seriously considering removing all references to MPI_Comm in nrn/src/nrniv nvector_parallel.* and nvector_nrnparallel_ld.*  . These are basically copies of sundials files from version 2 and 3 but modified to allow
construction from within nrn/src/nrncvode/cvodeobj.cpp and also work without directly calling anything from within mpi.h. That is, they are valid for ```-DNRN_ENABLE_MPI_DYNAMIC= ON or OFF and don't involve any mpi.h structures (such as MPI_Comm) or functions. They only call things available from ```nrn/src/nrnmpi/nrnmpi.h```.  I left in the MPI_Comm tokens just as indicators of what might have to be changed in future movment toward Sundials version 7.  At the moment, all of SUNDIALS used by neuron is inside libnrniv.so